editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -154,14 +154,21 @@ void SpriteFramesEditor::_sheet_add_frames() {
 
 	int fc = frames->get_frame_count(edited_anim);
 
+	Ref<AtlasTexture> atlas_source = Object::cast_to<AtlasTexture>(*split_sheet_preview->get_texture());
+
+	Rect2 region_rect = Rect2(0, 0, 0, 0);
```

Comment:
`Rect2` is default initialized to `Rect2(0, 0, 0, 0)` (via its `Vector2()` components), so you don't have to be explicit here.